base/services/wuauserv/wuauserv.c

```diff
@@ -0,0 +1,130 @@
+/*
+* PROJECT:          ReactOS Services
+* LICENSE:          GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:          Automatic Updates service stub.
+*                   Stub is required for some application at the installation phase.
+* COPYRIGHT:        Copyright 2018 Denis Malikov (filedem@gmail.com)
+*/
+
+#include "wuauserv.h"
+
+/* GLOBALS ******************************************************************/
+
+static WCHAR ServiceName[] = L"wuauserv";
+
+static SERVICE_STATUS_HANDLE ServiceStatusHandle;
+static SERVICE_STATUS ServiceStatus;
+
+static HANDLE exitEvent = NULL;
+
+/* FUNCTIONS *****************************************************************/
+
+static VOID
+UpdateServiceStatus(DWORD dwState)
+{
+    ServiceStatus.dwServiceType = SERVICE_WIN32_OWN_PROCESS;
+    ServiceStatus.dwCurrentState = dwState;
+    ServiceStatus.dwControlsAccepted = 0;
+    ServiceStatus.dwWin32ExitCode = 0;
+    ServiceStatus.dwServiceSpecificExitCode = 0;
+    ServiceStatus.dwCheckPoint = 0;
+
+    if (dwState == SERVICE_START_PENDING ||
+        dwState == SERVICE_STOP_PENDING ||
+        dwState == SERVICE_PAUSE_PENDING ||
+        dwState == SERVICE_CONTINUE_PENDING)
+        ServiceStatus.dwWaitHint = 10000;
```

Comment:
10 seconds seems a bit excessive for a dummy service?

Reply:
I look that almost all services have this value in UpdateServiceStatus() . Even empty stubs. What value should be here?

Reply:
It is the time the service expects to be able to move from pending to the actual state, so 1 or 5 seconds should be better. (the hint is in millisecond)